src/tools/Avalonia.Designer.HostApp/Avalonia.Designer.HostApp.csproj

```diff
@@ -14,6 +14,9 @@
     <Compile Include="..\..\..\src\Markup\Avalonia.Markup.Xaml.Loader\CompilerExtensions\**\*.cs" />
     <Compile Include="..\..\..\src\Markup\Avalonia.Markup.Xaml.Loader\AvaloniaXamlIlRuntimeCompiler.cs" />
     <Compile Include="..\..\..\src\Markup\Avalonia.Markup.Xaml.Loader\CompilerDynamicDependencies.cs" />
+    <Compile Include="..\..\..\src\Avalonia.Base\Metadata\NullableAttributes.cs" Link="Compatibility\NullableAttributes.cs" />
+    <Compile Include="..\..\..\src\Avalonia.Base\Compatibility\TrimmingAttributes.cs" Link="Compatibility\TrimmingAttributes.cs" />
+    <Compile Include="..\..\..\src\Shared\IsExternalInit.cs" Link="Compatibility\IsExternalInit.cs" />
```

Comment:
We need to unify how these pollyfils are referenced. Somewhere it's part of Avalonia.Base + internals visible to, and somewhere it's a shared file.

Reply:
Not sure what's better. But shared file is more annoying when you already have internals visible to enabled.

Reply:
:) 
<img width="638" alt="Screenshot 2024-05-20 at 9 10 47 PM" src="https://github.com/AvaloniaUI/Avalonia/assets/3163374/d2f97a19-f807-4966-9061-ad2f9e1d8ae3">